hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/KeyInputStream.java

```diff
@@ -264,7 +264,7 @@ protected synchronized int readWithStrategy(ByteReaderStrategy strategy)
 
       // Get the current blockStream and read data from it
       BlockExtendedInputStream current = blockStreams.get(blockIndex);
-      int numBytesToRead = Math.min(buffLen, (int)current.getRemaining());
+      int numBytesToRead = (int)Math.min(buffLen, current.getRemaining());
```

Comment:
Above line needs format based on latest checkstyle rules. Please rebase based on latest EC branch, which got updated with latest rules and it might flag warning here.

Reply:
Thanks for your reminding, rebased and waiting for CI.